lib/qs/payload_handler.rb

```diff
@@ -70,12 +70,12 @@ def raise_if_debugging!(exception)
     end
 
     def log_received
-      log_verbose "===== Running job ====="
+      log_verbose "===== Received message ====="
     end
 
-    def log_job(job)
-      log_verbose "  Job:     #{job.name.inspect}"
-      log_verbose "  Params:  #{job.params.inspect}"
+    def log_message(message)
+      log_verbose "  Name:    #{message.route_id.inspect}"
+      log_verbose "  Params:  #{message.params.inspect}"
```

Comment:
@kellyredding - I wouldn't mind doing specific logging but I was thinking it would require a conditional and didn't want to do that.

I was just thinking though that I could do like the payload module and use `send` here. Then I could have `log_job` and `log_event`. This would only work for the verbose logging and not the summary logging though.

This would allow the following for events:

```
Channel: a-channel
Name:    an-event
Params:  ...
```

I can make part of this change now adding the `send`, `log_job` and `log_event` methods. The event method will temporarily be logging an event job not an event which will have to be changed once its an actual event. Or I can make the change later once event is a kind of message.